config/centos/default/grr-http-server

```diff
@@ -1,5 +0,0 @@
-# Configuration file for grr-http-server start-up script
```

Comment:
I actually put those files there in the first place back in Oct just as storage so people could find server startup scripts that would work with Centos/RHEL if they ran the servers on non-Ubuntu Linux.  The rpmbuilder is only dealing with the client stuff so it only cares about  grr-client.initd.in